.circleci/config.yml

```diff
@@ -122,8 +122,8 @@ jobs:
           name: Link the module being tested to the samples.
           command: |
             cd samples/
-            npm install
             npm link @google-cloud/language
```

Comment:
this could be `npm link ../` instead, and then you wouldn't have to run `npm link` in the root dir.

Reply:
What is the preferred way (if any)?

Reply:
does not matter much, the way I mentioned just requires fewer steps.